demisto_sdk/commands/lint/helpers.py

```diff
@@ -282,8 +282,8 @@ def get_python_version_from_image(image: str) -> float:
     """
     docker_client = docker.from_env()
     py_num = 2.7
```

Comment:
since more and more integration and scripts are using python3, should we switch the fallback to python3?

Reply:
since most are on 3.8, ill set it to that